lametro/templates/lametro/index.html

```diff
@@ -88,15 +88,15 @@ <h2>
             </div>
         </div>
         <div class="row-fluid">
-            <div class='col-sm-10 col-sm-offset-1'>
+            <div class='col-sm-10 col-sm-offset-1' id='past-meetings'>
```

Comment:
Would you mind using double quotes for consistency?

Reply:
No problem.